### PR TITLE
darts-api environment files

### PR DESCRIPTION
**Pull request by scott-robertson1**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DMP-43

### Change description ###

the creation of the environment files for the darts-api

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by thomast1906 — CHANGES_REQUESTED** [review]
image is duplicated through, is all planned to use prod image? 
If so, remove reference from each patch file as its referenced already in app base https://github.com/hmcts/sds-flux-config/blob/6fa545bedfffbce7f529b4ce370bfbfabe381421/apps/darts-modernisation/darts-api/darts-api.yaml#L11

File: apps/darts-modernisation/darts-api/demo.yaml [hunk 1]
```diff
@@ -8,4 +8,4 @@ spec:
   values:
     java:
       ingressHost: darts-api.demo.platform.hmcts.net
-      image: sdshmctspublic.azurecr.io/darts/api:prod-272ebc2-20230322092842 # {"$imagepolicy": "flux-system:darts-api-prod"}
```
Comment (jackmaloney) [inline apps/darts-modernisation/darts-api/demo.yaml]: Do we not want to leave the image line in there for some of these files, so we can override the version that is deployed to a given environment at some point?

Comment (scott-robertson1) [reply apps/darts-modernisation/darts-api/demo.yaml]: can do, just demo for now ?

File: apps/darts-modernisation/darts-api/demo.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+kind: HelmRelease
+metadata:
+  name: darts-api
+  namespace: darts-modernisation
+spec:
+  releaseName: darts-api
+  values:
+    java:
+      ingressHost: darts-api.demo.platform.hmcts.net
+      image: sdshmctspublic.azurecr.io/darts/api:prod-272ebc2-20230322092842 # {"$imagepolicy": "flux-system:darts-api"}
```
Comment (thomast1906) [inline apps/darts-modernisation/darts-api/demo.yaml:11]: This is referenced here https://github.com/hmcts/sds-flux-config/blob/c0b984151aecb0525e18ccc3ce8d60e0db98e69f/apps/darts-modernisation/darts-api/darts-api.yaml

**Review by thomast1906 — APPROVED** [review]

**Review by jackmaloney — APPROVED** [review]

**Pull request merged**